sound/soc/sof/topology.c

```diff
@@ -1695,19 +1632,10 @@ int sof_load_pipeline_ipc(struct device *dev,
 	struct snd_sof_dev *sdev = dev_get_drvdata(dev);
 	int ret;
 
-	mutex_lock(&sdev->cores_status_mutex);
-
-	ret = sof_core_enable(sdev, pipeline->core);
```

Comment:
wouldn't it be possible to begin the series with the introduction of `*_get()` / `*_put()` functions to avoid adding code to be deleted immediately again?

Reply:
> wouldn't it be possible to begin the series with the introduction of `*_get()` / `*_put()` functions to avoid adding code to be deleted immediately again?

good point, let me rearrange them.